### PR TITLE
fix(curriculum): external userstories in interview prep

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

clear the 17th task in #40749

----

I wonder if is there something I can do to the codepen links and how to treat them going forward

### CODE REVIEW

**Comment by Sboonny** [comment]
Camper bot you think if I did a mistake I won't do it again😅, that's why you got scouted gj camperbot 😂😂😂

**Review by moT01 — COMMENTED** [review]
I think we might want to keep the codepen links for now @Sboonny. Users still need to use it to create these projects for now. I would be fine removing the wiki link. I think since there's a number of challenges that have this structure, I would prefer to add them all to one PR. They're all the "projects" - there's a number of these 'take home projects' on that list, as well as 'data visualization' and 'frontend libraries' projects.

**Comment by Sboonny** [comment]
@moT01 if you notice a file I missed, I happy to change it.

---

for contest I change make a link app, because the solution doesn't require the apps to be made in these sites, and to stop people from thinking they are limited to these sites

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ dashedName: build-a-25--5-clock
 
 # --description--
 
-**Objective:** Build a [CodePen.io](https://codepen.io) app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/XpKrrW>.
+**Objective:** Build an app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/XpKrrW>.
 
-Fulfill the below [user stories](https://en.wikipedia.org/wiki/User_story) and get all of the tests to pass. Give it your own personal style.
+Fulfill the below user stories. Use whichever libraries or APIs you need. Give it your own personal style.
```
Comment (moT01) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md:13]: I would keep the "and get all of the tests to pass" on all the projects that have automated tests.

```suggestion
Fulfill the below user stories and get all of the tests to pass. Use whichever libraries or APIs you need. Give it your own personal style.
```

**Review by moT01 — CHANGES_REQUESTED** [review]
I left one suggestion below. Additionally, I think we should remove all the `**User Stories**` titles that were added. They look a little out of place in cases:

Screen Shot 2022-05-26 at 10 39 36 AM

Also, that title wasn't added on some of the projects cause they didn't have a codepen link or whatever you used to search for which projects to change.

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md (hunk 1 above)
Comment (Sboonny) [reply curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md:13]: Sure I am on it

**Review by lynxlynxlynx — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Pull request merged**